lib/hashie/extensions/indifferent_access.rb

```diff
@@ -29,6 +29,7 @@ def self.included(base)
         base.class_eval do
           alias_method :regular_writer, :[]= unless method_defined?(:regular_writer)
           alias_method :[]=, :indifferent_writer
+          alias_method :_regular_writer, :[]=
```

Comment:
For the new usage of _regular_writer in initialize_attributes, this method has to exist, and this serves to alias it to the indifferent writer, otherwise we get both the string and symbol versions of the key added to the dash.